dll/win32/kernel32/kernel32.spec

```diff
@@ -1138,7 +1138,7 @@
 @ stdcall UnregisterWait(long)
 @ stdcall UnregisterWaitEx(long long)
 @ stub -version=0x600+ UpdateCalendarDayOfWeek
-@ stub -version=0x600+ UpdateProcThreadAttribute
+@ stdcall -stub -version=0x600+ UpdateProcThreadAttribute(ptr long ptr ptr ptr ptr ptr)
```

Comment:
Third/Fifth parameter are DWORD_PTR/SIZE_T, I'd argue they should be a `long`? 🤷

Reply:
Oh I didn't realize `long` would not print the top bits. Yes, totally agree with you then.